### PR TITLE
chore: Bumped packages using `npm run bump-all-packages`

### PR DESCRIPTION
**Pull request by heath-freenome**
### Reasons for making this change

Bumped all packages to their latest

### Checklist

- [ ] **I'm updating documentation**
  - [ ] I've [checked the rendering](https://rjsf-team.github.io/react-jsonschema-form/docs/contributing) of the Markdown text I've added
- [ ] **I'm adding or updating code**
  - [ ] I've added and/or updated tests. I've run `npm run test:update` to update snapshots, if needed.
  - [ ] I've updated [docs](https://rjsf-team.github.io/react-jsonschema-form/docs) if needed
  - [ ] I've updated the [changelog](https://github.com/rjsf-team/react-jsonschema-form/blob/main/CHANGELOG.md) with a description of the PR
- [ ] **I'm adding a new feature**
  - [ ] I've updated the playground with an example use of the feature

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,31 +30,32 @@
...
-    "eslint-plugin-jsx-a11y": "^6.7.1",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jsx-a11y": "^6.8.0",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "husky": "^8.0.3",
-    "jest-environment-jsdom": "^29.6.4",
+    "jest-environment-jsdom": "^29.7.0",
+    "jest-environment-jsdom-global": "^4.0.0",
```
Comment (nickgros) [inline package.json:53]: @heath-freenome why was this new dependency added?

File: packages/semantic-ui/package.json [hunk 2]
```diff
@@ -39,33 +39,34 @@
...
+    "babel-jest": "^29.7.0",
     "esbuild": "^0.18.20",
-    "eslint": "^8.48.0",
-    "jest": "^29.6.4",
-    "jest-environment-jsdom": "^29.6.4",
-    "nanoid": "^3.3.6",
+    "eslint": "^8.56.0",
+    "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
+    "jest-environment-jsdom-global": "^4.0.0",
```
Comment (nickgros) [inline packages/semantic-ui/package.json:63]: ditto

Comment (heath-freenome) [reply packages/semantic-ui/package.json:63]: Since this is in the main `package.json` I can remove this one

File: package.json (hunk 1 above)
Comment (heath-freenome) [reply package.json:53]: It was needed at some point to fix an issue with tests. And removing it now seems to not cause any problems.

**Review by nickgros — APPROVED** [review]

**Pull request merged**